cli/lib/kontena/cli/plugins/uninstall_command.rb

```diff
@@ -6,10 +6,10 @@ class UninstallCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::PluginManager::Common
 
-    parameter 'NAME ...', 'Plugin name', attribute_name: :plugins
+    parameter 'NAME ...', 'Plugin name', attribute_name: :plugins_to_uninstall
 
     def execute
-      plugins.each do |name|
+      plugins_to_uninstall.each do |name|
```

Comment:
Hmm, shouldn't that have been `plugins_list` anyway?

Reply:
Changed to stick to the standard clamp `name_list` convention.